template/flake.nix

```diff
@@ -0,0 +1,19 @@
+{
+  description = "virtual environments";
+
+  inputs.devshell.url = "github:numtide/devshell";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, flake-utils, devshell }:
+    flake-utils.lib.eachDefaultSystem (system: {
+      devShell = let shell = import devshell { inherit system; }; in
```

Comment:
Most likely the user will want to control the version of nixpkgs as well.

something like that:
```nix
let
  pkgs = import nixpkgs { overlays = [ devshell.overlay ]; inherit system; };
in
{
  devShell = pkgs.devshell.fromTOML ./devshell.toml;
}
```